docs/development/getting-started.md

```diff
@@ -126,7 +126,7 @@ them has died.
 You can find the names of the services (used in `docker-compose build api`)
 in the `docker-compose.yml` file. At the time of writing (2021-05-10), those are
 `proxy`, `api`, `provider-gitlab`, `provider-github`, `provider-azuredevops`,
-`db`, `rabbitmq`
+`db`
```

Comment:
Suggest also updating the date, as at the previous date (2021-05-10) the `rabbitmq` service still existed

(I can't seem to add a GitHub review suggestion here, weird)